web/yo/app/scripts/controllers/gene.js

```diff
@@ -3589,6 +3599,7 @@ angular.module('oncokbApp')
                         'Sorry for any inconvinience.');
                 }
                 $scope.fileEditable = false;
+                $scope.geneEditable = $scope.fileEditable;
```

Comment:
We should. Having a watcher is not a bad practice. It's actually designed to deal with the situation like this.